### PR TITLE
[DOC] Updates the TEDPCA section following changes in tedana v0.0.8

### LINKED ISSUE #547: More documentation for --tedpca
### Summary

Between version **0.0.7** and **0.0.8**, the `--tedpca` parameter changed.
Is it planned to enhance the documentation explaining which _tedpca_ does what ?

### Additional Detail

The "decision tree" part in the code is proably the most difficult part 

Help in **0.0.7** :
```
Possible choices: mle, kundu, kundu-stabilize
Method with which to select components in TEDPCA
Default: “mle”
```

Help in **0.0.8** :
```
Possible choices: kundu, kundu-stabilize, mdl, aic, kic
Method with which to select components in TEDPCA. PCA decomposition with the mdl, kic and aic options is based on a Moving Average (stationary Gaussian) process and are ordered from most to least aggresive. Default=’mdl’.
Default: “mdl”
```

### Next Steps

Explain a bit more what does each decision tree ?
In particular, why "mdl" is choose to be the default one ?

### PR DESCRIPTION
**Pull request by eurunuela**
Closes #547.

Changes proposed in this pull request:

- Removes the MLE description from the TEDPCA section in the docs.
- Adds the MA-PCA description to the TEDPCA section in the docs.

### CODE REVIEW

**Comment by dowdlelt** [comment]
I like that the documentation is getting this update - definitely needed for consistency. However, I think the "simply put" section should be a little more non math-wizard friendly. Maybe if there was just a sentence that stated the goal of this process (aka, this method is used to more optimally select the number of components for fMRI data, which typically violates a number of statistical assumptions? if that is even right....). 

I do like the following section that makes it clear what each option does, I just can't follow the stat jargon and feel like I might not be alone in that.

**Comment by eurunuela** [comment]
Totally understand your point. I'll make it simpler. The reader could always check the paper anyway.

Thanks for the feedback @dowdlelt!

**Comment by eurunuela** [comment]
How about now @dowdlelt?

File: docs/approach.rst [hunk 1]
```diff
@@ -208,10 +208,18 @@ Here we can see time series for some example components (we don't really care ab
...
 
 These components are subjected to component selection, the specifics of which
-vary according to algorithm.
+vary according to algorithm. Speficically, ``tedana`` offers two different approaches that perform this step.
 
-In the simplest approach, ``tedana`` uses Minka’s MLE to estimate the
-dimensionality of the data, which disregards low-variance components (the `mle` option in for `--tedpca`).
+The default approach (the `mdl` option for `--tedpca`) is based on a Moving Average (stationary Gaussian) process
+proposed by `Li et al (2007)`_. A moving average process is the output of a linear system (which in this case is
+a smoothing filter) that has an i.i.d. Gaussian process as the input. If we assume that the linear system is shift
```
Comment (jbteves) [inline docs/approach.rst]: Could we clarify what i.i.d. is here?

Comment (eurunuela) [reply docs/approach.rst]: Sure, I’ll use the complete name. Thanks for the feedback! 🙌🏻

File: docs/approach.rst [hunk 2]
```diff
@@ -208,10 +208,18 @@ Here we can see time series for some example components (we don't really care ab
 .. image:: /_static/a11_pca_component_timeseries.png
 
 These components are subjected to component selection, the specifics of which
-vary according to algorithm.
+vary according to algorithm. Speficically, ``tedana`` offers two different approaches that perform this step.
 
-In the simplest approach, ``tedana`` uses Minka’s MLE to estimate the
-dimensionality of the data, which disregards low-variance components (the `mle` option in for `--tedpca`).
+The default approach (the `mdl` option for `--tedpca`) is based on a Moving Average (stationary Gaussian) process
```
Comment (tsalo) [inline docs/approach.rst]: This _isn't_ just the `mdl` option, though.

File: docs/approach.rst [hunk 3]
```diff
@@ -208,10 +208,18 @@ Here we can see time series for some example components (we don't really care ab
...
+proposed by `Li et al (2007)`_. A moving average process is the output of a linear system (which in this case is
+a smoothing filter) that has an independent and identically distributed Gaussian process as the input. If we assume that the linear system is shift
+invariant, the moving average process is a stationary Gaussian random process. Simply put, this process more optimally
+selects the number of components for fMRI data following a subsampling scheme described in `Li et al (2007)`_. The
+selection of components is performed with either of the three options provided by `--tedpca`:
+
+* `aic`: the Akaike Information Criterion, which is the least aggresive option; i.e. returns the biggest amount of components.
```
Comment (tsalo) [inline docs/approach.rst]:
```suggestion
* `aic`: the Akaike Information Criterion, which is the least aggressive option; i.e., returns the largest number of components.
```

**Comment by eurunuela** [comment]
Thanks for the feedback @tsalo! Let me know if it's clearer now ;)

**Comment by eurunuela** [comment]
Thank you guys! I'll merge this.

**Comment by eurunuela** [comment]
Okay guys, for some reason my latest commit was not added into the PR... I have no idea why. It only changed the following line:

> The default approach (the mdl option for --tedpca) is...

I'll open a new PR to add this change. It should be a quick one to merge.

**Pull request merged**